### PR TITLE
Test - Graphql Query tests

### PR DESCRIPTION
**Pull request by FrancisBourgouin**
## 🚀Changes

Added tests for :
- archipelagos
- archipelago by archipelagoId & islanderId
- island by islandId & islanderId
- islander by islanderId

## Checklist

- [ X ] Prefixed the PR name with the convention **Feat**, **Fix**, **Test**, **Chore**
- [ X ] Ran the linter with `yarn lint` or `yarn lint --fix`
- [ X ] Ensure the changes pass tests

### CODE REVIEW

File: backend/__tests__/queries.test.js [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+          name
+          nativeFruit
+          islanders{
+            id
+            name
+          }
+        }
+      } 
+    }`;
+		const expectedResponse = {
```
Comment (DerangedMind) [inline backend/__tests__/queries.test.js:84]: Maybe we could move this variable definition to a `before()` or to an outer scope? It's reused a few times throughout and would help keep the code DRY

File: backend/__tests__/queries.test.js [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+      island(islanderId: 3){
+        id
+        name
+        nativeFruit
+        islanders{
+          id
+          name
+        }
+      }
+    }`;
```
Comment (DerangedMind) [inline backend/__tests__/queries.test.js:257]: Just noticed that your queries are all missing some indentations! Do you have format on save enabled? It's my favourite for fixing alignment 🤤

File: backend/__tests__/queries.test.js (hunk 1 above)
Comment (FrancisBourgouin) [reply backend/__tests__/queries.test.js:84]: The name expectedResponse is indeed used in each test, however the value is different. Wouldn't make sense

File: backend/__tests__/queries.test.js (hunk 2 above)
Comment (FrancisBourgouin) [reply backend/__tests__/queries.test.js:257]: Yeah indentation is onSave, I think it's because of the +... added by github when you review it

**Pull request merged**